app/src/main/java/org/mozilla/fenix/trackingprotection/TrackingProtectionPanelView.kt

```diff
@@ -107,7 +108,7 @@ class TrackingProtectionPanelView(
     private fun setUIForNormalMode(state: TrackingProtectionState) {
         details_mode.visibility = View.GONE
         normal_mode.visibility = View.VISIBLE
-        protection_settings.isGone = state.session?.customTabConfig != null
+        protection_settings.isGone = state.tab?.source == SessionState.Source.CUSTOM_TAB
```

Comment:
@csadilek is this the right way to identify a custom tab?

Reply:
When I tested it worked but I just want to double check

Reply:
Yeah I saw that and had to look it up too. It seems safe as we always set the source when create the `CustomTabSessionState.` However, you can also write `state.tab is CustomTabSessionState` instead which we use in all other places.

Reply:
👍🏽